src/main/java/Duke.java

```diff
@@ -6,5 +14,167 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("\t------------------------------------------");
+        System.out.println("\tHello there! I'm Duke.");
+        System.out.println("\tWhat can I help you with?");
+        System.out.println("\t__________________________________________\n");
+
+        ArrayList<Task> tasks = new ArrayList<>();
+        boolean runLoop = true;
+
+        while (runLoop) {
+            String userInput = sc.nextLine().trim();
+
+            if (userInput.equalsIgnoreCase("bye")) {
+                exitProgram();
+                runLoop = false;
+            } else if (userInput.equalsIgnoreCase("list")) {
+                listTasks(tasks);
+            } else if (userInput.split(" ")[0].equalsIgnoreCase("done")) {
+                markTask(userInput, tasks);
+            }
+            else if (userInput.split(" ")[0].equalsIgnoreCase("delete")) {
```

Comment:
Perhaps you could avoid complicated (and repeated) expressions by breaking it down into multiple lines and storing it as a variable first; I noticed the same issue in a few other classes too, and it would help a reader understand your code better!